components/containers/Admin/Organisations/OrganisationSettings.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Fragment } from "react";
+import { useTranslation } from "next-i18next";
+import Head from "next/head";
+import { Organisation } from "@lib/types";
+import { DeleteButton } from "../../../forms/Button/DeleteButton";
+import axios from "axios";
+
+interface OrganisationSettingsProps {
+  organisation: Organisation;
+}
+
+const handleDelete = async (organisationID: string) => {
+  // redirect to view templates page on success
+  const resp = await axios({
+    url: "/api/organisations",
+    method: "POST",
+    headers: {
+      "Content-Type": "multipart/form-data",
+    },
+    data: {
+      method: "DELETE",
+      organisationID: organisationID,
+    },
+    timeout: 0,
+  })
+    .then((serverResponse) => {
+      //success - redirect to view-templates page
+      return serverResponse;
+    })
+    .catch((err) => {
```

Comment:
We might want to test this pattern of catching the error.  I just noticed this pattern in a couple of places.
- If there's an error in axios it'll get caught in the `.catch(err)` block.
- The error is then logged to console.   (We should probably migrate these to logMessage.error eventually)
- The body of the error is then returned to `resp` which returns the key `status`.  My concern if that if the `status` key doesn't exist in the error object then `undefined` is being returned.  
- I'm wondering if we should bubble up an error or explicitly return a `status: 500` to ensure the `status` key is always available.

I know the logic that calls this function explicitly checks for a `200` status code and if not send the error message to console again.  We can refactor this in a future PR as I know this is just to continue the work with the DB in staging.

Reply:
For now I'll just modify it to return the entire resp object if there is no status. The DeleteButton component is looking specifically for a return of exactly 200, so if anything else is passed it will just error out anyways.

We can refactor this better in the future as needed.